components/common/Icons.tsx

```diff
@@ -153,41 +155,38 @@ export const CircularIonicon = ({
           circularStyles.iconBackground,
           {
             backgroundColor,
-            width: DEFAULT_ICON_SIZE * 1.5,
-            height: DEFAULT_ICON_SIZE * 1.5
+            width: size * 1.5,
+            height: size * 1.5
```

Comment:
```suggestion
            width: size * useFontScale({ maximumFontSizeMultiplier: maximumFontScaleFactor }) * 1.5,
            height: size * useFontScale({ maximumFontSizeMultiplier: maximumFontScaleFactor }) *  1.5
```
Without this, you get this when the user uses an accessibility font size.

![IMG_0151](https://github.com/user-attachments/assets/e303818c-5206-4ce1-9ba1-a58474d142b2)